modules/oidcsso/src/test/java/org/wso2/carbon/tomcat/oidcsso/extension/TestConstants.java

```diff
@@ -0,0 +1,104 @@
+    public static final String INVALID_DESCRIPTOR = "faulty-xml.xml";
+    // test constants for oidc configuration.
+    public static final String CLIENT_ID = "abdefghijklm_123415";
+    public static final String CLIENT_SECRET = "6789nopqrstuvwxyz";
+    public static final String REDIRECT_URI = "http://localhost:8080/foo-app/openid";
+    public static final String SCOPE = "openid,profile";
+    public static final String CLAIMS = "address";
+    public static final String RESPONSE_TYPE = "code";
+    public static final String GRANT_TYPE = "authorization_code";
+    public static final String AUTHENTICATION_ENDPOINT = "https://localhost:9443/oauth2/authorize";
+    public static final String TOKEN_ENDPOINT = "https://localhost:9443/oauth2/token";
+    public static final String USER_INFO_ENDPOINT = "https://localhost:9443/oauth2/userinfo?schema=openid";
+    public static final String LOGOUT_ENDPOINT = "https://localhost:9443/oidc/logout";
+    //  test constants for server level security configurations.
+    public static final String TYPE = "JKS";
+    public static final String PRIVATE_KEY_ALIAS = "wso2carbon";
+    public static final String TRUSTSTORE_PATH = "${catalina.base}/conf/wso2/client-truststore.jks";
+    public static final String TRUSTSTORE_PASSWORD = "wso2carbon";
+    public static final String KEYSTORE_PATH =
+            "src/test/resources/testResources/oidcExtension/conf/wso2/wso2carbon.jks";
+    // Authentication Response Constants.
+    public static final String CODE = "8e0e20bb-4e33-3ea6-b931-66eb5a73d9d3";
+    public static final String STATE = "hello";
+    public static final String REQUEST_SCOPE = "openid,email";
+    public static final String INVALID_SCOPE = "profile";
+    public static final String SESSION_STATE = "25960f5fd9c7c4c3b4d07c776bc5feb5ac9b8752861057f01.mWZHKZC3k9vc7JnOGCU";
+    public static final String ERROR = "invalid_request";
+    public static final String AUTHENTICATION_REQUEST = "https://localhost:9443/oauth2/authorize?display=popup&" +
+            "response_type=code&client_id=abdefghijklm_123415&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Ffoo-app" +
+            "%2Fopenid&scope=openid+email&state=hello&claims=%7B%22userinfo%22%3A%7B%22address%22%3Anull%7D%7D";
+    public static final String AUTHENTICATION_RESPONSE_URL = SSL_PROTOCOL + "://" + DEFAULT_TOMCAT_HOST + ":" +
+            SSL_PORT + "/" + SAMPLE_WEB_APP + "/openid";
+    public static final String INVALID_AUTHENTICATION_RESPONSE_URL = DEFAULT_TOMCAT_HOST + ":" +
+            SSL_PORT + "/" + SAMPLE_WEB_APP + "/openid";
+    public static final String SAMPLE_STRING = "abckjacdcdkjnascs";
+    public static final String AUTHENTICATION_RESPONSE_QUERY = "code=" + CODE + "&state=" + STATE + "&session_state=" +
+            SESSION_STATE;
+    public static final String INVALID_AUTHENTICATION_RESPONSE_QUERY = "state=" + STATE + "&session_state=" +
```

Comment:
INVALID_AUTHENTICATION_RESPONSE_QUERY has been used in OIDCAgentTest .java in line number 159